gst/nnstreamer/tensor_common.c

```diff
@@ -668,6 +713,7 @@ gst_tensor_config_from_structure (GstTensorConfig * config,
  * @brief Get tensor caps from tensor config
  * @param config tensor config info
  * @return caps for given config
+ * @todo Add layout
```

Comment:
Discussion: About caps negotiation, is it necessary to indicate data format in caps string?
I think the data format is used only in tensor-filter now, for invoking specific nn-framework like nnfw and snap.

Reply:
In a pipeline with multiple tensor filters (of nnfw or snap), I think having data format/layout in caps might be helpful. It would stop the connection between two tensor filters with same dimensions but different data format (which I think is desirable).
For caps negotiation between data layout/formats, I think the only 1 pair will mis-match - `(NCHW, NHWC)`. All other pairings (even with NONE layout as NONE layout represents `dont care` or `unknown layout`) should match. (This isn't implemented yet). Please provide your input on this.

Reply:
I am not sure to do the negotiation with new field 'layout' is possible. (field-type is string and it should be exactly matched)
@helloahn @again4you IMO, this may require huge change in Tizen ACR process. Please see the details.

Reply:
@jaeyun-jung Yes, negotiation with string should match exactly. Removed layout from caps.